DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202205061441)]
+    public class SeparateCentreEmailsTable : Migration
+    {
+        public override void Up()
+        {
+            Create.Table("UserCentreDetails")
+                .WithColumn("ID").AsInt32().NotNullable().PrimaryKey().Identity()
+                .WithColumn("UserID").AsInt32().NotNullable().ForeignKey("Users", "ID")
+                .WithColumn("CentreID").AsInt32().NotNullable().ForeignKey("Centres", "ID")
+                .WithColumn("Email").AsString(255).Nullable()

```

Comment:
Should this really be nullable? I know the ticket states that it should be but I would expect this to not be nullable, and any accounts that don't have an email just don't have a record. Unless we expect this table to have a load more columns in the future @SteveJacksonSoft?

Reply:
Yeah, we don't know whether there will be more details on this table, but it's possible.
Given that, I think it's best to leave this nullable.